app/controllers/staff/transactions_controller.rb

```diff
@@ -52,6 +52,18 @@ def update
     end
   end
 
+  def destroy
+    @transaction = Transaction.find(id)
+    authorize @transaction
+
+    @transaction.create_activity key: "transaction.destroy", owner: current_user
```

Comment:
What's the effect of creating public_activity records for something you then delete? Do the logs remain present, and if so how does one view them?

Reply:
I don't think we can view `public_activity` records for anything can we? This is more so we have an audit trail that we can get at right? Or am I missing something? (I frequently do!)

Reply:
I could potentially add parameters to the Activity, to make it clearer what has been deleted?